esx_service/vmodl/vmodl_test.py

```diff
@@ -690,7 +690,7 @@ def test_update_privilege_with_invalid_volume_size(self):
 
         # Update the privilege with invalid volume size
         with self.assertRaises(vmodl.fault.InvalidArgument):
-            self.tenantMgr.UpdatePrivilege(tenant, self.datastore, volume_max_size=2048, volume_total_size=1024)
+            self.tenantMgr.UpdatePrivilege(tenant, self.datastore, allow_create=True, volume_max_size=2048, volume_total_size=1024)
```

Comment:
Since python VMODL for VCS UpdatePrivilege is using None value for allow_create parameter if it is not provided. This is invalid; unless we have a default behavior when the value for allow_create option is None or not provided during tenant_access_set procedure.